cdc/processor/pipeline/cyclic_mark.go

```diff
@@ -174,3 +181,39 @@ func extractReplicaID(markRow *model.RowChangedEvent) uint64 {
 	log.Panic("bad mark table, " + mark.CyclicReplicaIDCol + " not found")
 	return 0
 }
+
+// cyclicNodeContext implements the NodeContext, cyclicMarkNode can be reused in table actor
+// buffer all messages with a queue, it will not block the actor system
+type cyclicNodeContext struct {
+	*actorNodeContext
+	queue list.List
+}
+
+func NewCyclicNodeContext(ctx *actorNodeContext) *cyclicNodeContext {
+	return &cyclicNodeContext{
+		actorNodeContext: ctx,
+	}
+}
+
+// SendToNextNode implement the interface function, push the message to queue
```

Comment:
may be `SendToNextNode implements NodeContext, ...` is better?

Reply:
done